README.md

```diff
@@ -1,7 +1,7 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-Portfolio
+Trail Portfolio Project
 
 > Description the project.
 This a portfolio trials project.
```

Comment:
Hi @Tshepo45👌 ,Indeed you did an exceptional job but there are a few things that need to be addressed to go to next level.
The project name has typo errors, please consider adding a professional title and description to your ReadMe.md file as it will make your project professional .

Let's do it again, Champ! 💪